advocacy_docs/partner_docs/EsriArcGISPro/04-ConfiguringEsriArcGISPro.mdx

```diff
@@ -0,0 +1,218 @@
+
+The test cases in this guide are using the sample 'HongKong GDB' data file for demonstration, which can be found here: https://www.arcgis.com/home/item.html?id=96a56c14f72c4638961f4ee9a271e34d.
+
+## Enable Environment to Create an Enterprise Geodatabase
+
+1. Generate an authorization file for Esri ArcGIS Server license to generate a keycodes file for your Enterprise Geodatabase according to Esri docs.
+
+2. Verify that you have the authorization file which is typically located in `C:\Program Files (x86)\Common Files\ArcGIS\bin`.
+
+3. Double-click `SoftwareAuthorization.exe` to run the authorization wizard to start generating your keycodes file.
+
+4. In the Software Authorization Wizard, under `Authorization Options`, select `I have received an authorization file and am now ready to finish the authorization process`.
+
+  ![EsriAuthorization](Images/EsriAuthorization.png)
+
+5. Click `Browse...` to select the .ecp authorization file obtained from My Esri in Step 2.
+
+6. For the Product to be Authorized, ensure ArcGIS Server is selected, and click `Next`.
+
+7. On the `Authorization Method` page, accept the default option, `Authorize with Esri now using the Internet`, and click `Next`.
+
+  ![EsriAuthorizationMethodScreen](Images/EsriAuthorizationMethodScreen.png)
+
+8. On the `Authorization Information` page, confirm that your contact information is correct, click `Next`.
+
+9. On the `Authorization Information (continued)` page enter your industry information and confirm your selections, click `Next`.
+
+  ![EsriAuthorizationChoicesNew](Images/EsriAuthorizationChoicesNew.png)
+
+10. Provide your license number on the `Software Authorization Number` screen and click `Next`.
+
+11. Select the default option, `I do not want to authorize any extensions at this time` on the `Authorize Software Extensions` page if you do not want any extensions.
+
+  ![EsriAuthorizationExtensions](Images/EsriAuthorizationExtensions.png)
+
+12. Click `Next` and the authorization process will start, when it has finished you will receive a message that states the software has been authorized.
+
+13. Click `Finish` when your license has been successfully authorized, and this will have created your keycodes file.
+
+14. Navigate to the following location to access the newly created keycodes file and verify the keycodes file is present: `C:\Program Files(x86)\Esri\License10.x\sysgen`.
+
+15. This keycodes file will be used when you create an Enterprise Geodatabase.
+
+## Esri ArcGIS Pro Configuration for Enterprise Sharing
+
+Before you can share from your Esri ArcGIS Pro environment to an Enterprise Portal that you have created, you will need to add that Portal in your Esri ArcGIS Pro environment.
+ 
+ 1. Click `Projects` then select `Portals`.
+ 2. Click `Add Portal` to add the URL of the Portal you wish to use.
+ 3. Set the Portal as the active portal by selecting the option from the three dots to the right.
+ 4. Right click the Portal and select `Sign in`.
```

Comment:
Should we then say they are now setup for sharing or something?
Seems incomplete as it is now.

Reply:
"After you have signed in to your Esri ArcGIS Enterprise Portal you will then be ready to share to an Esri ArcGIS Enterprise environment."

Reply:
I'm trying to use "an" instead of "the" for these cases to differentiate that a customer doesn't have to share to Enterprise.